_docs/1.0.0/usage-schema.md

```diff
@@ -139,7 +139,7 @@ You provide *Schema* to **Standardization** in a JSON file:
 }
 ```
 
-Example of data adhering to the above schema can be found [here](https://github.com/AbsaOSS/enceladus/blob/develop/standardization/src/test/scala/za/co/absa/enceladus/standardization/samples/TestSamples.scala).
+Example of data adhering to the above schema can be found [here](https://github.com/AbsaOSS/enceladus/blob/master/spark-jobs/src/test/scala/za/co/absa/enceladus/standardization/samples/TestSamples.scala).
```

Comment:
Could I be a *** and ask you to update the URL to `https://github.com/AbsaOSS/enceladus/blob/v1.7.1/standardization/src/test/scala/za/co/absa/enceladus/standardization/samples/TestSamples.scala`

I checked. There is no more master or develop URL issues in the whole gh-pages branch

Reply:
Sure, done.

No need to be calling yourself any colorful names, though. Neither ***, nor ****** ***** ****, or even !@#$%^& 🤣

Reply:
@Zejnilovic  Yeah, we know them all anyway 😈 
@dk1844 Thanks for being the [good boy scout](https://deviq.com/boy-scout-rule/) 🏅

Reply:
@benedeki more like [this](https://youtu.be/dQw4w9WgXcQ)